continuous_integration/install.sh

```diff
@@ -47,7 +47,15 @@ if [[ "$DISTRIB" == "conda" ]]; then
 
 elif [[ "$DISTRIB" == "ubuntu" ]]; then
     # Use standard ubuntu packages in their default version
-    sudo apt-get install -qq python-scipy python-nose python-pip
+    sudo apt-get install -qq python-scipy
+    # At the time of writing numpy 1.9.1 is included in the travis
+    # virtualenv but we want to used numpy installed through apt-get
+    # install.
+    deactivate
+    # Create a new virtualenv using system site packages for numpy and scipy
+    virtualenv --system-site-packages testvenv
+    source testvenv/bin/activate
+    pip install nose
```

Comment:
Either you use the virtualenv and you have to install nosetests in it, or you don't use it. In this case we could have just disabled the venv provided by travis (with `deactivate`) and use the system `nosetests` command from `/usr/bin/nosetests`.

Travis uses a set of venvs by default to make it easy to test with different versions of Python, including versions that are not shipped by ubuntu 12.04.